src/eredis_sub.erl

```diff
@@ -14,16 +14,26 @@
 
 -export([psubscribe/2, punsubscribe/2]).
 
+-export_type([sub_options/0, sub_option/0]).
+
+-type sub_option() ::
+        {max_queue_size, integer() | infinity} |
+        {queue_behaviour, drop | exit} |
+        option().
+-type sub_options() :: [sub_option()].
+
 %%
 %% PUBLIC API
 %%
 
 start_link() ->
     start_link([]).
 
+%% @deprecated Use {@link start_link/1} instead.
```

Comment:
Since this is now deprecated maybe we should use the preferred `start_link` in eredis_pubsub_SUITE.erl

Reply:
Sure, but it's not important IMO.